compiler.rst

```diff
@@ -419,7 +461,27 @@ Important Files
 
     asdl_c.py
         "Generate C code from an ASDL description."  Generates
-        :file:`Python/Python-ast.c` and :file:`Include/Python-ast.h`.
+        :file:`Python/Python-ast.c` and :file:`Include/Internal/pycore_ast.h`.
+
+    parser.c
+        The parser generated by :file:`Tools/peg_generator/pegen/c_generator.py`
+        from the grammar :file:`Grammar/python.gram`.  Creates the AST from
+        source code.  Rule functions for their corresponding production rules
+        are found here.
+
+    peg_api.c
+        Contains high-level functions to create an AST from source code which
+        are used by the interpreter.
+
+    pegen.c
+        The new PEG parser introduced in 3.9.  Contains helper functions which
```

Comment:
Let's move the first sentence, `The new PEG parser introduced in 3.9.` to `parser.c`. `pegen.c` only contains helper functions.